multiload/properties.c

```diff
@@ -652,9 +652,8 @@ fill_properties(GtkWidget *dialog, MultiloadApplet *ma)
 	gtk_container_set_border_width (GTK_CONTAINER (page), 12);
 	add_color_selector(page, _("_User"), "memload-color0", ma);
 	add_color_selector(page, _("Sh_ared"), "memload-color1", ma);
-	add_color_selector(page, _("_Buffers"), "memload-color2", ma);
-	add_color_selector (page, _("Cach_ed"), "memload-color3", ma);
-	add_color_selector(page, _("F_ree"), "memload-color4", ma);
+	add_color_selector(page, _("Cach_ed"), "memload-color2", ma);
```

Comment:
Key `memload-color2` is used here .......

Reply:
This is in an old commit!